lib/src/ast/css/media_query.dart

```diff
@@ -194,31 +203,33 @@ class CssMediaQuery {
 ///
 /// This is either the singleton values [empty] or [unrepresentable], or an
 /// instance of [MediaQuerySuccessfulMergeResult].
-abstract class MediaQueryMergeResult {
-  /// A singleton value indicating that there are no contexts that match both
-  /// input queries.
-  static const empty = _SingletonCssMediaQueryMergeResult("empty");
+class MediaQueryMergeResult {
```

Comment:
I'd rather keep this as-is rather than split it into a nested class if new enums can't support extra data on a subtype. Same with `ColorFormat`.

Reply:
yup, those two were a bit difficult to wrangle